components/clamav/files/etc/clamav-milter.conf.example-v0.98.1

```diff
@@ -0,0 +1,293 @@
+##
```

Comment:
Sorry, old *standard* config sample files slipped in from in-house legacy ;)
Still, they did not change in upstream sources since the specified versions.

Fixed away.